Added randomnumber command + added new code to main.py

zFlippy opened the pull request:
Added the randomnumber command and I added an exception handler to main.py because of the situation where the user includes anything other than numbers in the random number command.

zFlippy committed:
Updated main.py

Added exception handler because of the new randomnumber command.

zFlippy committed:
Added randomnumber command.

plexthedev reviewed (changes requested):
Delete the error handler in main.py, instead use a try/except statement in the part of the random number generator where you convert the numbers to integers. It would also be nice if you could clean the code a little, especially the empty space spanning lines 4-6

zFlippy committed:
Updated main.py

Removed exception handler

zFlippy committed:
Updated randomnumber.py

Added try: except: function and removed the blank lines that were there for some reason
